pro/src/components/pages/Offerers/Offerer/VenueV1/VenueEdition/ImageVenueUploaderSection/VenueImageUploaderModal.tsx

```diff
@@ -46,6 +46,10 @@ export const VenueImageUploaderModal = ({
   const [editedImage, setEditedImage] = useState('')
   const [isUploading, setIsUploading] = useState(false)
   const [editorInitialScale, setEditorInitialScale] = useState(1)
+  const [editorInitialPosition, setEditorInitialPosition] = useState({
+    x: 0.5,
+    y: 0.5,
+  })
```

Comment:
J'ai l'impression qu'on à un meme usage ici.
Il semble qu'on ai pas besoin du type vu que ton code tourne. Je me demande si c'est une bonne pratique de le spécifier.
Un avis ?

```suggestion
  const [editorConfig, setEditorConfig] = useState<{
    scale: number,
    position: Position,
  }>({
    scale: 1,
    position: { x: 0.5,  y: 0.5 },
  })
```

Reply:
Bonne suggestion, il y a sûrement moyen d'utiliser une même variable pour la position et le scale

Reply:
On va faire une session de mob avec Hugo et Bruno pour améliorer la gestion de la scale et de la position et on ouvrira une autre PR